### PR TITLE
[Dominion] Externalize component content

### PR DESCRIPTION
**Pull request by kilemensi**
## Description

Organize and externalize country content from components.

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Screenshots

Screenshot

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: dominion/dominion_ui/src/components/ProfileReleases/index.js [hunk 1]
```diff
@@ -84,15 +84,8 @@ const styles = theme => ({
...
-  const { profileDataJson } = window;
-  let primaryReleases;
-  if (
-    profileDataJson &&
-    Object.prototype.hasOwnProperty.call(profileDataJson, 'primary_releases')
-  ) {
-    primaryReleases = profileDataJson.primary_releases;
-  }
+function ProfileReleasesSection({ classes, datasetReleases }) {
+  console.log(`RELEASES ${JSON.stringify(datasetReleases)}`);
```
Comment (karimkawambwa) [inline dominion/dominion_ui/src/components/ProfileReleases/index.js]: remove log

**Review by karimkawambwa — CHANGES_REQUESTED** [review]

File: dominion/dominion_ui/src/index.js [hunk 2]
```diff
@@ -20,16 +20,24 @@ import Partners from './components/Partners';
 import ProfileTabs from './components/ProfileTabs';
 import Showcase from './components/Showcase';
 import Video from './components/Video';
-import ProfileReleasesSection from './components/ProfileReleases';
+import ProfileReleases from './components/ProfileReleases';
 
 library.add(fab);
 
+const PROPS = {
+  countries: window.all_countries || {},
```
Comment (karimkawambwa) [inline dominion/dominion_ui/src/index.js]: `all_countries ` => `dominion_countries`

**Review by karimkawambwa — CHANGES_REQUESTED** [review]

File: dominion/templates/settings_js.html [hunk 3]
```diff
@@ -2,16 +2,18 @@
...
   {% endif %}
   var geography = {{ geography.this |jsonify|safe }};
-  var all_countries = Object.entries({{ HURUMAP.countries | jsonify| safe }});
-  var selected_country = all_countries.find((item) => item[1].code === geography.geo_code.slice(0,2))[1];
   {% if country %}
-    selected_country = {{ country|jsonify|safe}}
+    var selected_country = {{ country|jsonify|safe}}
+  {% else %}
+    var selected_country = Object
+      .entries({{ HURUMAP.countries | jsonify| safe }})
```
Comment (karimkawambwa) [inline dominion/templates/settings_js.html]: can't you just use `countries` here?

**Review by karimkawambwa — CHANGES_REQUESTED** [review]

File: dominion/dominion_ui/src/components/ReleaseDropdown.js [hunk 4]
```diff
@@ -123,17 +124,17 @@ class ReleaseDropdown extends Component {
           open={menuOpen}
           onClose={this.closeReleaseMenu}
         >
-          <Paper>
-            <ClickAwayListener onClickAway={this.closeReleaseMenu}>
+          <ClickAwayListener onClickAway={this.closeReleaseMenu}>
+            <Paper>
```
Comment (karimkawambwa) [inline dominion/dominion_ui/src/components/ReleaseDropdown.js:128]: Did this order solve the click away issue ? @kilemensi

Comment (kilemensi) [reply dominion/dominion_ui/src/components/ReleaseDropdown.js:128]: Yep @karimkawambwa

**Review by karimkawambwa — APPROVED** [review]
LGTM
Godly PR works.
Just resolve merge conflict.

File: dominion/dominion_ui/src/components/ReleaseDropdown.js (hunk 4 above)
Comment (karimkawambwa) [reply dominion/dominion_ui/src/components/ReleaseDropdown.js:128]: Tested it 👍

**Comment by karimkawambwa** [comment]
😅 one more conflict

**Pull request merged**